cpu/esp8266/ld/esp8266.riot-os.ld

```diff
@@ -114,8 +114,8 @@ SECTIONS
     /* TODO put only necessary .rodata to dram */
     /* *(.rodata .rodata.*) */
     *libc.a:*.o(.rodata.* .rodata)
-    *core.a:*(.rodata.* .rodata)
-    *cpu.a:*(.rodata .rodata.*)
+    *core/*(.rodata.* .rodata)
```

Comment:
There is another instance of `core.a` on line 223

Reply:
well, I missed that but it's a comment...